deps/patches/llvm-3.9-osx-10.12.patch

```diff
@@ -0,0 +1,33 @@
+--- a/projects/compiler-rt/cmake/builtin-config-ix.cmake
++++ b/projects/compiler-rt/cmake/builtin-config-ix.cmake
```

Comment:
is this path always present in the llvm tarball, or can this patch only be applied if clang/compiler-rt is being built?